TMGToolbox/src/analysis/transit/estimate_transit_operating_costs.py

```diff
@@ -0,0 +1,1045 @@
+        pb.branding_text = "TMG Toolbox"
+        pb.add_select_scenario(tool_attribute_name='Scenario', title='Scenario:', allow_none=False,
+                               note="Select the scenario to perform the cost computation for. Be sure to have the time "
+                                    "period information (AM, MD, PM, EV, or ON) present in the scenario title. "
+                                    "For time periods other than ON, please select a scenario with completed transit "
+                                    "assignment results for better accuracy.")
+        pb.add_select_file(tool_attribute_name='ServiceTableFile', title='Service Table File:', file_filter="*.csv",
+                           window_type='file',
+                           note="The Service Table file contains three columns with a header row: emme_id, trip_depart,"
+                                " trip_arrive. emme_id is the Emme transit line ID, and trip_depart and trip_arrive "
+                                "are the start and end times of each scheduled transit trip in HH:MM:SS.")
+        pb.add_select_file(tool_attribute_name='CostParamsFile', title='Cost Parameters File:', file_filter="*.csv",
+                           window_type='file',
+                           note="The Cost Parameters file contains ten columns with a header row: "
+                                "<ul><li>agency_prefix: one or two capital letters</li>"
+                                "<li>mode: the ID of the mode the set of parameters is for</li>"
+                                "<li>uc_revhr: unit cost per revenue hour</li>"
+                                "<li>uc_revkm: unit cost per revenue km</li>"
+                                "<li>uc_veh_annual: annual unit cost per peak vehicle</li>"
+                                "<li>uc_veh_daily: daily unit cost per peak vehile. Only one of uc_veh_annual and "
+                                "uc_veh_daily is required.</li>"
+                                "<li>weekday_ratio_revhr: multiplier to get annual estimates of revenue hours from a "
+                                "weekday daily value</li>"
+                                "<li>weekday_ratio_revkm: multiplier to get annual estimates of revenue km from a "
+                                "weekday daily value</li>"
+                                "<li>adj_revhr: calibrated adjustment factor for revenue hours</li>"
+                                "<li>adj_revkm: calibrated adjustment factor for revenue km</li></ul>")
+        pb.add_select_file(tool_attribute_name='ReportFile', title="Report File:", file_filter="*.csv",
+                           window_type='save_file')
+        # pb.add_select_file(tool_attribute_name='DebugFile', title="Debug File:", file_filter="*.csv",
+        #                    window_type='save_file')
+        if self.tool_run_msg:
+            pb.add_html(self.tool_run_msg)

```

Comment:
It would be good to move this to the position after line 91 so that the tool completion message shows up on the top.